test/metabase/search/appdb/index_test.clj

```diff
@@ -53,15 +53,14 @@
                         :model/Card       {}            {:name "Projected Satisfaction"         :collection_id col-id#}
                         :model/Database   {db-id# :id}  {:name "Indexed Database"}
                         :model/Table      {}            {:name "Indexed Table", :db_id db-id#}]
-           (search.index/reset-index!)
-           (search.ingestion/populate-index! :search.engine/appdb)
+           (search.engine/reindex! :search.engine/appdb {:in-place? true})
```

Comment:
Creating a new index feels cleaner to me, like starting up the app again. Curious as to your reasoning, but don't think the distinction is too important in practice.